specs/xaml-backdrop-api.md

```diff
@@ -0,0 +1,316 @@
+
+
+Xaml Backdrop APIs
+===
+
+# Background
+
+Composition in WinAppSDK has a general
+[ISystemBackdropController](https://docs.microsoft.com/windows/windows-app-sdk/api/winrt/Microsoft.UI.Composition.SystemBackdrops.ISystemBackdropController)
+interface that's concretely implemented by
+[MicaController](https://docs.microsoft.com/windows/windows-app-sdk/api/winrt/Microsoft.UI.Composition.SystemBackdrops.MicaController)
+and [DesktopAcrylicController](https://docs.microsoft.com/windows/windows-app-sdk/api/winrt/Microsoft.UI.Composition.SystemBackdrops.DesktopAcrylicController).
+A backdrop is a visual effect you can use as the background for your window (or island).
+For example the Desktop Acrylic backdrop lets you set as the background of your window the desktop wallpaper behind it, but visually modified.
+[More info](https://learn.microsoft.com/en-us/windows/apps/design/style/acrylic).
+
+These existing Mica and Desktop Acrylic controllers allow you to set the backdrops onto a
+[WindowId](https://docs.microsoft.com/windows/windows-app-sdk/api/winrt/Microsoft.UI.WindowId),
+and there's currently only an awkward way to get a `WindowId` from a Xaml object such as
+[Window](https://docs.microsoft.com/windows/windows-app-sdk/api/winrt/Microsoft.UI.Xaml.Window).
+
+The awkwardness only begins there; you also need to work with a _system_
+[DispatcherQueue](https://docs.microsoft.com/uwp/api/Windows.System.DispatcherQueue),
+coordinate with the
+[Loaded](https://docs.microsoft.com/windows/windows-app-sdk/api/winrt/Microsoft.UI.Xaml.FrameworkElement.Loaded)
+event, and check
+[IsSupported](https://docs.microsoft.com/windows/windows-app-sdk/api/winrt/Microsoft.UI.Composition.SystemBackdrops.MicaController.IsSupported)
+to see if you need to provide a fallback (Mica isn't supported on Windows 10).
+
+So it's a great feature and works, but it's difficult to use.
+[Here's a full code example of the current solution](https://learn.microsoft.com/en-us/windows/apps/windows-app-sdk/system-backdrop-controller#example-use-mica-in-a-windows-appsdkwinui-3-app).
+
+The purpose of the new APIs here are to make backdrops much easier to use in a Xaml app,
+allowing you to simply set a backdrop as a property on a `Window`.
+
+
+# API Pages
+
+## Window.SystemBackdrop property
+
+_This description also applies to `Popup.SystemBackdrop` and `FlyoutBase.SystemBackdrop` properties_
```

Comment:
Should we also mention DesktopWindowXamlSource.SystemBackdrop?

Reply:
Recommend: Yes. (DWXS is experimental, but so is Popup and FlyoutBase)